### ISSUE TITLE
Error during version comparison

### ISSUE THREAD

**Issue by gongled** [essentialkaos/ek#114]
### Bug report

I have been tried to use version package and compare two versions between each other.

**System info:**

* **Version used:** ek.v8
* **OS:** Mac OS 10.12.4
* **Kernel:** Darwin Kernel Version 16.5.0
* **Go version:** 1.8.1

**System info:**

Have a look at the following source code:
```
    var v1, v2 version.Version

    v1, _ = version.Parse("2.0.0")
    v2, _ = version.Parse("1.1.0")

    fmtc.Println(v1.String(), v2.String(), v1.Greater(v2))

    v1, _ = version.Parse("1.1.0")
    v2, _ = version.Parse("2.0.0")

    fmtc.Println(v1.String(), v2.String(), v1.Greater(v2))
```

The result of execution is:
```
2.0.0 1.1.0 false
1.1.0 2.0.0 false
```

**Expected behavior:**

I expect that `2.0.0` is greater than `1.1.0`.

**Actual behavior:**

It returns false.

**Comment by andyone** [comment]
Fixed in #115. Can you please check this version?

**Comment by gongled** [comment]
Thank you! Now it works perfectly.
